tailwind.config.js

```diff
@@ -3,10 +3,12 @@ module.exports = {
     extend: {
       colors: {
         'dark-blue': 'rgb(21, 35, 71)',
+        'dark-blue-alt': '#1d2c4e',
         'light-blue': '#92EAFF',
         'mid-blue': '#39486E',
         'mid-purple': '#A11EC6',
-        'mid-grey': '#99a1b3'
+        'mid-grey': '#99a1b3',
+        yellow: '#fff922'
```

Comment:
Extremely minor but can we wrap `yellow` in quotes as well?

Reply:
All sorted!